app/src/main/res/layout/fragment_promotions.xml

```diff
@@ -15,33 +15,52 @@
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         >
-      <androidx.recyclerview.widget.RecyclerView
-          android:id="@+id/rv_promotions"
+      <FrameLayout
+          android:id="@+id/top_half"
           android:layout_width="match_parent"
           android:layout_height="wrap_content"
-          android:layout_marginTop="@dimen/big_margin"
           android:orientation="vertical"
-          android:overScrollMode="never"
-          app:layoutManager="androidx.recyclerview.widget.LinearLayoutManager"
-          app:layout_constraintEnd_toEndOf="parent"
-          app:layout_constraintStart_toStartOf="parent"
           app:layout_constraintTop_toTopOf="parent"
-          tools:itemCount="1"
-          tools:listitem="@layout/item_promotions_gamification"
-          />
+          >
+        <androidx.recyclerview.widget.RecyclerView
+            android:id="@+id/rv_promotions"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/big_margin"
+            android:orientation="vertical"
+            android:overScrollMode="never"
+            app:layoutManager="androidx.recyclerview.widget.LinearLayoutManager"
+            app:layout_constraintTop_toTopOf="parent"
+            tools:itemCount="1"
+            tools:listitem="@layout/item_promotions_gamification"
+            />
+
+        <include
+            android:id="@+id/locked_promotions_with_vouchers"
+            layout="@layout/layout_locked_promotions_with_vouchers"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="32dp"
```

Comment:
I think this top margin should be a bit smaller, to be better in accordance with design